src/Bicycle.java

```diff
@@ -0,0 +1,22 @@
+public class Bicycle extends Transport {
+
+
+    public Bicycle(String modelName, int wheelsCount) {
+       super(modelName, wheelsCount);
+    }
+
+
+    public void update() {
+        System.out.println("Меняем покрышки " + getWheelsCount());
```

Comment:
чтобы был виден полиморфизм надо указать конкретную реализацию
```suggestion
        System.out.println("Меняем покрышки велосипеда " + getWheelsCount());
```